src/Open3D/Visualization/Rendering/Filament/PointCloudBuffers.cpp

```diff
@@ -24,19 +24,18 @@
 // IN THE SOFTWARE.
 // ----------------------------------------------------------------------------
 
-#include "FilamentGeometryBuffersBuilder.h"
-
-#include "FilamentEngine.h"
-#include "FilamentResourceManager.h"
-#include "Open3D/Geometry/BoundingVolume.h"
-#include "Open3D/Geometry/PointCloud.h"
-
-#include <Eigen/Core>
-
 #include <filament/IndexBuffer.h>
 #include <filament/VertexBuffer.h>
 #include <filament/geometry/SurfaceOrientation.h>
 
```

Comment:
delete empty line?

Reply:
clang-format inserted the blank line. check-style fails without it.

Reply:
this turns out to be a difference between the way clang-format-5 and clang-format-9 implement SortIncludes. Empty line removed.